src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -237,6 +289,35 @@ private string ClearHexString(string hexString)
             }
         }
 
+        /// <summary>
+        /// Converts a BigInteger to hex string to UTF8 string
+        /// input:  860833102
+        /// output: NEO3
+        /// </summary>
+        /// <param name="strParam">
```

Comment:
@chenzhitong could you fix these warnings?

Reply:
Already modified